### PR TITLE
Update integration.rst

### PR DESCRIPTION
**Pull request by colinele**
et là ça marche

### CODE REVIEW

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

File: documentation/doc_sncf/source/integration.rst [hunk 1]
```diff
@@ -111,7 +102,7 @@ All the resources return a response containing a links object, a paging object,
 +---------------------------------------------------------------+-------------------------------------+
 
 
-Authentication
+Authentification
```
Comment (antoine-de) [inline documentation/doc_sncf/source/integration.rst]: in english it's authentication :smile:

File: documentation/doc_sncf/source/integration.rst [hunk 2]
```diff
@@ -466,21 +460,12 @@ the selected public transport object or coordinates.
...
-|                                                                                                                                                             |
-| When using the journeys endpoint the data set used to compute the journey is chosen using the possible datasets of the origin and the destination.          |
-|                                                                                                                                                             |
-| For the moment it is not yet possible to compute journeys on different data sets, but it will one day be possible (with a cross-data-set system).           |
-|                                                                                                                                                             |
-| If you want to use a specific data set, use the journey api within the data set: `<https://api.navitia.io/v1/coverage/{your_dataset}/journeys>`_            |
+| The API SNCF handle to compute journey from "Stop Area" to " Stop Area" or "Stop Area" to "Administrative Region"            |
```
Comment (antoine-de) [inline documentation/doc_sncf/source/integration.rst]: I'm not sure, but I think it's more english to say:

"The SNCF API handle journey computation from "

As for the content, I think what you mean is that you cannot do coordinates to coordinates computation no ?

File: documentation/doc_sncf/source/integration.rst [hunk 3]
```diff
@@ -713,7 +618,7 @@ tags                array of string    List of tags on the journey. The tags add
 +-------------------------+------------------------------------+----------------------------------------------------+
 | geojson                 | `GeoJson <http://www.geojson.org>`_|                                                    |        
 +-------------------------+------------------------------------+----------------------------------------------------+
-| path                    | Array of path_                     | The path of this section                           |
+|                         |                                    | tracé sur la carte en collier de perle             |
```
Comment (antoine-de) [inline documentation/doc_sncf/source/integration.rst]: english :)

File: documentation/doc_sncf/source/integration.rst [hunk 4]
```diff
@@ -996,6 +843,12 @@ color           string                 Color of the line
 routes          array of `route`_      Routes of the line
 commercial_mode `commercial_mode`_     Commercial mode of the line
 =============== ====================== ============================
++-----------------------------------------------------------------------------------------------------------+
+| *Note*                                                                                                    |
+|                                                                                                           |
+| The fiels "Code" and "Color" in this API are not available                                                |
```
Comment (antoine-de) [inline documentation/doc_sncf/source/integration.rst]: fields

File: documentation/doc_sncf/source/integration.rst [hunk 5]
```diff
@@ -996,6 +843,12 @@ color           string                 Color of the line
...
+| The lines you get with API do not correspond to commercial line but only to origin and destination mode   |
```
Comment (kinnou02) [inline documentation/doc_sncf/source/integration.rst]: not clear, we will look at it together

File: documentation/doc_sncf/source/integration.rst [hunk 6]
```diff
@@ -1056,6 +909,11 @@ name             string                          Name of the commercial mode
 physical_modes   array of `physical_mode`_       Physical modes of this commercial mode
 ================ =============================== =======================================
 
++-----------------------------------------------------------------------------------------------------------+
+| *Note*                                                                                                    |
+| As commercial mode you will find in the API SNCF (TGV, TER, Intercité, Transilien)                                                                                                         |
```
Comment (antoine-de) [inline documentation/doc_sncf/source/integration.rst]: you mean to list the commercial modes here ?

If so I think you should maybe be more explicit:

```
The commercial modes available in the SNCF API are:
 * ...
 * ...
```

File: documentation/doc_sncf/source/integration.rst (hunk 2 above)
Comment (colinele) [inline documentation/doc_sncf/source/integration.rst]: yes indeed

File: documentation/doc_sncf/source/integration.rst (hunk 3 above)
Comment (colinele) [inline documentation/doc_sncf/source/integration.rst]: Street network !! is that right Sir ?

File: documentation/doc_sncf/source/integration.rst (hunk 5 above)
Comment (colinele) [inline documentation/doc_sncf/source/integration.rst]: voui

File: documentation/doc_sncf/source/integration.rst [hunk 7]
```diff
@@ -342,10 +380,31 @@ Other examples
 	* https://api.navitia.io/v1/coverage/fr-idf/physical_modes/physical_mode:Metro/lines
 
 
-Places
+6.2.2. Specific parameters
+###################
+
+There are som specific parameters.
+A least, there is one: *odt_level* which can be applied only on /lines collection...
```
Comment (kinnou02) [inline documentation/doc_sncf/source/integration.rst]: We won't have any odt in the sncf data

File: documentation/doc_sncf/source/integration.rst [hunk 8]
```diff
@@ -1,29 +1,117 @@
...
+This document describes how to use the SNCF API based on navitia via the v1 interface, and the returned resources.
+Navitia is an Open Source software developed by Canal TP. (www.canaltp.fr)
+
+The SNCF API handle to :
+* compute journey from and to "station" or "administrative region" 
+* display next departures or arrivals
+* display route schedules
+* lexicographical search (autocomplete)
+
+The SNCF API contain theorical train data for the following commercial modes : TGV, TER, Transilien, Intercités.
```
Comment (antoine-de) [inline documentation/doc_sncf/source/integration.rst]: contains

**Comment by antoine-de** [comment]
you have some problems with the RST formating, as you can see here:
https://github.com/colinele/navitia/blob/patch-1/documentation/doc_sncf/source/integration.rst

File: documentation/doc_sncf/source/integration.rst [hunk 9]
```diff
@@ -1,29 +1,60 @@
...
+.. contents:: Index
 
 Overview
 ========
 
-This document describes how to call navitia via the v1 interface, and the returned resources.
+This document describes how to use the SNCF API based on navitia via the v1 interface, and the returned resources.
+Navitia is an Open Source software developed by Canal TP. (www.canaltp.fr)
+
+The SNCF API handle to :
```
Comment (antoine-de) [inline documentation/doc_sncf/source/integration.rst:11]: I think it better to say:

```
The SNCF API handles:

* journeys from and to "station" or "administrative region"
* next departures or arrivals
* route schedules
* lexicographical search (autocomplete)
```

there is no stop schedule ?

File: documentation/doc_sncf/source/integration.rst [hunk 10]
```diff
@@ -1,29 +1,60 @@
...
 
-* Geographical coverage of the service
-	* https://api.navitia.io/v1/coverage 
-* Where am I? (WGS 84 coordinates)
-	* https://api.navitia.io/v1/coord/2.377310;48.847002
-	* I'm on the "/fr-idf" coverage, at "20, rue Hector Malot in Paris, France"
+* Transport mode available in the service
+	* http://api.sncf.com/v1/coverage/sncf/
 * Which services are available on this coverage? take a look at the links under this URL
 	* https://api.navitia.io/v1/coverage/fr-idf
```
Comment (TeXitoi) [inline documentation/doc_sncf/source/integration.rst:54]: Urls are not modified to use the sncf.com domain and sncf examples.

File: documentation/doc_sncf/source/integration.rst [hunk 11]
```diff
@@ -1,29 +1,60 @@
...
+Read our lexicon : https://github.com/OpenTransport/vocabulary/blob/master/vocabulary.md
+
+Authentication
+================
+
+You must authenticate to use **SNCF API**. When you register we give you an authentication key to the API.
+
+There is two ways for authentication:
+
+You can use a `Basic HTTP authentication`_, where the username is the key, and without password.
```
Comment (antoine-de) [inline documentation/doc_sncf/source/integration.rst:29]: you need more blank lines

File: documentation/doc_sncf/source/integration.rst [hunk 12]
```diff
@@ -1,29 +1,60 @@
...
+The SNCF API contains theorical train data for the following commercial modes : TGV, TER, Transilien, Intercités.
+
+Read our lexicon : https://github.com/OpenTransport/vocabulary/blob/master/vocabulary.md
+
+Authentication
+================
+
+You must authenticate to use **SNCF API**. When you register we give you an authentication key to the API.
+
+There is two ways for authentication:
```
Comment (antoine-de) [inline documentation/doc_sncf/source/integration.rst:27]: "There is two ways for the authenticate" no ? (not sure at all there, but seems strange to me)

File: documentation/doc_sncf/source/integration.rst [hunk 13]
```diff
@@ -461,37 +444,18 @@ Journeys
...
-|                                                                                                                                                             |
-| When using the journeys endpoint the data set used to compute the journey is chosen using the possible datasets of the origin and the destination.          |
-|                                                                                                                                                             |
-| For the moment it is not yet possible to compute journeys on different data sets, but it will one day be possible (with a cross-data-set system).           |
-|                                                                                                                                                             |
-| If you want to use a specific data set, use the journey api within the data set: `<https://api.navitia.io/v1/coverage/{your_dataset}/journeys>`_            |
+| The API SNCF handle computation journey from and to "station" or "administrative region"                                                                    |
```
Comment (antoine-de) [inline documentation/doc_sncf/source/integration.rst:455]: "journey computation" I think no ?

**Comment by antoine-de** [comment]
apart from my last remarks it's ok for me.

**Pull request merged**